src/components/modules/paste.ts

```diff
@@ -439,6 +439,13 @@ export default class Paste extends Module {
   private handlePasteEvent = async (event: ClipboardEvent): Promise<void> => {
     const { BlockManager, Toolbar } = this.Editor;
 
+    if (
+      !(event.target instanceof Element) ||
+      !event.target.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)
+    ) {
+      return;
+    }
+
```

Comment:
And... This fix is also resolved by https://github.com/codex-team/editor.js/pull/1431
This PR is invalid. 
Sorry, I'll close it!